spec/indexers/cypripedium_indexer_spec.rb

```diff
@@ -21,13 +21,13 @@
 
       let(:attrs) {
         { title: ['My Title'],
-          date_created: ['1970-04'],
+          date_created: ['1970-04-30'],
           # creator: ['Kehoe, Patrick J.', 'Backus, David', 'Kehoe, Timothy J.'],
           creator_id: [creator_one.id, creator_two.id, creator_three.id] }
       }
       it 'indexes a sortable title and date created' do
         expect(solr_doc['title_ssi']).to eq 'My Title'
-        expect(solr_doc['date_created_ssi']).to eq '1970-04'
+        expect(solr_doc['date_created_ssi']).to eq '1970-04-30'
```

Comment:
I think you need to add a test to make sure `date_created_iti` is being indexed correctly.  Maybe something like this:
```
expect(solr_doc['date_created_iti']).to eq '1970'
```